brainiak/utils/utils.py

```diff
@@ -773,25 +774,58 @@ def p_from_null(X, two_sided=False):
         above the null) or two-sided (testing for both significantly positive
```

Comment:
You should update this docstring to indicate that X can either have this form, or contain only the real result if max/min arguments are used

Reply:
Thanks, I added some descriptions to the docsting